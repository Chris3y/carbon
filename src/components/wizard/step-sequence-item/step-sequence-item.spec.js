import React from 'react';
import { shallow } from 'enzyme';
import StepSequenceItem from './step-sequence-item';

describe('<StepSequenceItem />', () => {
  let stepSequenceItem;

  beforeAll(() => {
    stepSequenceItem = shallow(
      <StepSequenceItem
        state='foo'
        label='bar'
        stepNumber={ 2 }
      />
    );
  });

  test('basic render', () => {
    expect(stepSequenceItem).toMatchSnapshot();
  });

  test('completed render', () => {
    stepSequenceItem.setProps({ state: 'complete' });
    expect(stepSequenceItem).toMatchSnapshot();
  });
});
