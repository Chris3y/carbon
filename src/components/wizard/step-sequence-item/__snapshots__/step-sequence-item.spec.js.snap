// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StepSequenceItem /> basic render 1`] = `
<li
  className="carbon-step-sequence-item"
>
  <span>
    2
  </span>
  <span>
    bar
  </span>
</li>
`;

exports[`<StepSequenceItem /> completed render 1`] = `
<li
  className="carbon-step-sequence-item"
>
  <span>
    <Icon
      bgSize="small"
      type="tick"
    />
  </span>
  <span>
    bar
  </span>
</li>
`;
